types/openai.ts

```diff
@@ -7,7 +9,9 @@ export interface OpenAIModel {
 
 export enum OpenAIModelID {
   GPT_3_5 = 'gpt-3.5-turbo',
+  GPT_3_5_AZ = 'gpt-35-turbo',
```

Comment:
Hi. First, thanks for providing this reference implementation!

But note that Azure provides a way to name the model, what they call "deployment name" ([step 6 of this link)](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal#deploy-a-model):

> The model name will be used as the deployment name via OpenAI client libraries and API.

Therefore, we need a way to specify custom model names instead of hard-coding them.